res/css/views/rooms/_AppsDrawer.pcss

```diff
@@ -324,9 +324,12 @@ limitations under the License.
     display: flex;
     height: 100%;
     flex-direction: column;
-    justify-content: center;
     align-items: center;
+}
+
+.mx_AppWarning {
     font-size: $font-16px;
+    justify-content: center;
```

Comment:
This is because applying `justify-content: center` to `mx_AppPermission` makes it impossible to scroll to the top of the small pane.